layers/+lang/clojure/packages.el

```diff
@@ -4,11 +4,12 @@
     cider-eval-sexp-fu
     clj-refactor
     clojure-mode
+    clojure-snippets
```

Comment:
Maybe use `(clojure-snippets :toggle (configuration-layer/layer-usedp 'auto-completion)` to install package only if the `auto-completion` layer is installed